src/plugins/controls/public/options_list/components/options_list_strings.ts

```diff
@@ -48,9 +48,11 @@ export const OptionsListStrings = {
         defaultMessage: 'Popover for {fieldName} control',
         values: { fieldName },
       }),
-    getLoadingMessage: () =>
-      i18n.translate('controls.optionsList.popover.loading', {
-        defaultMessage: 'Loading options',
```

Comment:
A bunch of these `OptionsListStrings` were not being used - tried to clean up as many of them as I could find.